tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -63,12 +66,25 @@ def option_price(volatilities,
     forwards: A real `Tensor` of any shape that broadcasts to the shape of
       `volatilities`. The forwards to maturity. Either this argument or the
       `spots` must be supplied but both must not be supplied.
+    risk_free_rates: An optional real `Tensor` of same dtype as the
+      `volatilities`. If not None, discount factors are calculated using it:
+      e^(-rT), where r is the risk free rate. If not None, `spots` is supplied
+      and `cost_of_carries` is not, also used to compute forwards to expiry.
+      Default value: None, equivalent to r = 0 and discount factors = 1 when
+      discount_factors also not given.
+    cost_of_carries: An optional real `Tensor` of same dtype as the
+      `volatilities`. If not None, and `spots` is supplied, used to calculate
```

Comment:
Maybe be a bit more explicit what it means? Something like a sentence from Wikipedia? 
https://en.wikipedia.org/wiki/Cost_of_carry

'Expresses the cost of storing a physical commodity.'

Reply:
'Cost of storing a physical commodity, the cost of
      interest paid when long, or the opportunity cost, or the cost of paying
      dividends when short'? Or is that too long?